test/code/scxsystemlib/memory/memoryinstance_test.cpp

```diff
@@ -652,6 +787,37 @@ class MemoryInstance_Test : public CPPUNIT_NS::TestFixture
 #endif
     }
 
+    void testMemAvailable()
+    {
+#if defined(linux)
+        // available memory on linux platforms with Kernel 3.14+ should Taken from MemAvailable
+
+        const scxulong c_totalMemory = 3522864;
+        const scxulong c_free = 175844;
+        const scxulong c_buffer = 36;
+        const scxulong c_cached = 289148;
+        const scxulong c_memAvail = 2813432;
+
```

Comment:
Added the /proc/meminfo file directly and removed unnecessary variables, which are not required for test case.